File: 2-semestre/uri-online-judge/java/1008.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int numeroFun, horasTrab;
        double valorHora, salarioTotal;
        
        Scanner entrada = new Scanner(System.in);
        
        numeroFun = entrada.nextInt();
        horasTrab = entrada.nextInt();
        valorHora = entrada.nextDouble();
        salarioTotal = horasTrab * valorHora;
        
        System.out.printf("NUMBER = %d\n", numeroFun);
        System.out.printf("SALARY = U$ %.2f\n", salarioTotal);
    }
}

File: 2-semestre/uri-online-judge/c/1008.c
#include <stdio.h>

int main() {
    int numeroFun, horasTrab;
    double valorHora, salario;
    
    scanf("%d", &numeroFun);
    scanf("%d", &horasTrab);
    scanf("%lf", &valorHora);
    salario = horasTrab * valorHora;
    
    printf("NUMBER = %d\n", numeroFun);
    printf("SALARY = U$ %.2lf\n", salario);
    
    return 0;
}
